packages/path_provider/path_provider_linux/example/pubspec.yaml

```diff
@@ -9,8 +9,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  path_provider:
-    path: ../../path_provider
+  path_provider: ^1.6.10
 
```

Comment:
I believe we also need:
```
path_provider_linux:
    path: ../
```

Otherwise running the example (including on CI) won't be testing local changes (including changes that are part of a PR)